docs/design/design.md

```diff
@@ -103,28 +149,39 @@ There should be:
      - be a client certificate (`x509.ExtKeyUsageClientAuth`)
      - have the CN `system:kube-scheduler`
 
-`ca.crt` is also embedded in all the KubeConfig files.
+Please note that:
+
+1. `ca.crt` is embedded in all the KubeConfig files.
+2. If a given kubeconfig file exists, and its content is evaluated compliant with the above specs, the existing file will be used and the generation phase for the given kubeconfig skipped.
+3. if `kubeadm` is running in "ExternalCA" mode, all the required kubeconfig must be provided by the user as well, because  `kubeadm` cannot generate any of them by itself.
+4. in case of `kubeadm`  executed in the `--dry-run` mode, kubeconfig files are written in a temporary folder.
+
+### Generate Static Pod Manifest for control plane
+
+Common properties for the control plane components:
 
-### Phase 3: Bootstrap the control plane by using Static Pods
+- `HostNetwork: true` is present on all static pods since there is no network configured yet; accordingly 
+  -  `address`of the api server will be set to `127.0.0.1`
```

Comment:
What?

Reply:
Rephrased. Hope that now it is more clear